examples/framesjs-starter/app/debug/page.tsx

```diff
@@ -77,7 +77,7 @@ export default function Page({
         body: JSON.stringify({
           untrustedData: {
             fid: farcasterUser.fid,
-            url: frame.postUrl,
+            url: url,
```

Comment:
can you add to the type doc that this is the frame url

Reply:
looks like it's already there

Reply:
https://github.com/framesjs/frames.js/blob/main/docs/pages/reference/js/types.mdx#L83